tools/themis/src/rules.rs

```diff
@@ -95,20 +95,19 @@ pub fn has_debuggable_rust_version(workspace: &Workspace) -> anyhow::Result<()>
 
     let mut outliers = vec![];
     for pkg in &workspace.members {
-        let (rust_version, raw) = match pkg.raw["package"].get("rust-version") {
+        match &pkg.parsed.rust_version {
             Some(rust_version) => {
-                let raw = rust_version.as_str().unwrap();
-                (semver::VersionReq::parse(raw)?, raw)
+                if !rust_version.matches(&rust_toolchain) {
+                    outliers.push(Outlier {
+                        path: pkg.parsed.manifest_path.clone(),
+                        found: Some(format!("{}", rust_version)),
+                    });
+                }
+            }
+            None => {
+                // we can skip, since we have has_rust_version check
+                continue;
```

Comment:
Re-reading this bit of code, I think we can just remove the whole function? We check that rust-version is specified at all in `has_rust_version`, and it cant' actually be greated that what we have in rust-toolcahin, as that would break the build. 

I suggest sending a follow-up just removing this logic (but maybe I am missing something obvious here?)

Reply:
For future PRs which combine "do the same trivail change all over the place" with "do small non-trivial change in one place", consider more explicitly calling this structure out, so that  the interesting bit isn't missed. (eg, by adding a comment yourself on GitHub, or setting-up history for per-commit review.)

Reply:
> Re-reading this bit of code, I think we can just remove the whole function? We check that rust-version is specified at all in has_rust_version, and it cant' actually be greated that what we have in rust-toolcahin, as that would break the build.

Hmm... that might indeed be the case.  I'll investigate a bit.  I was actually thinking of doing something simpler and combining `has_rust_version` and `has_debuggable_rust_version`.  They seem to be performing similar checks.  

> For future PRs which combine "do the same trivail change all over the place" with "do small non-trivial change in one place", consider more explicitly calling this structure out, so that the interesting bit isn't missed. (eg, by adding a comment yourself on GitHub, or setting-up history for per-commit review.)

Thanks for the feedback.  I will keep it in mind in future.